packages/playground/assets/vite.config.js

```diff
@@ -1,5 +1,10 @@
 const path = require('path')
 
+const execRoot = process.cwd()
+const playgroundDir = execRoot.includes('playground')
+  ? path.resolve(__dirname, '..') // Allow running vite locally inside the folder.
+  : path.resolve(execRoot, 'packages/playground')
```

Comment:
This could be simplified if we assume Vite is always run by jest, but added it in case folks run vite manually from the `assets` folder when debugging, so that it always resolves as expected.